# Because of how coverage is run, the current directory is the first in
# sys.path. This is a problem for the tests, because they may import a module of
# the same name as a module in the current directory.
#
# NOTE @aignas 2023-06-05: we have to do this before anything from coverage gets
# imported.
diff --git a/coverage/__main__.py b/coverage/__main__.py
index ce2d8db..7d7d0a0 100644
--- a/coverage/__main__.py
+++ b/coverage/__main__.py
@@ -8,1 +8,2 @@
 import sys
+sys.path.append(sys.path.pop(0))
